3.6.0/Dockerfile

```diff
@@ -21,4 +21,4 @@ RUN curl -fSL -o $GN_FILE \
 COPY ./docker-entrypoint.sh /entrypoint.sh
```

Comment:
The Alpine dist does not include curl nor zip, by default. We need to add those, or the build will fail.

`RUN apk update && apk add curl zip`